doctr/io/elements.py

```diff
@@ -253,6 +256,79 @@ def synthesize(self, **kwargs) -> np.ndarray:
 
         return synthesize_page(self.export(), **kwargs)
 
+    def export_as_xml(self, file_title: str = 'docTR - XML export (hOCR)', **kwargs) \
+            -> Tuple[bytes, ET.ElementTree]:
+        """Export the page as XML
+
+        Args:
+            return_plain: whether to return the plain (bytes) XML string or an ElementTree object
+            file_title: the title of the XML file
+            **kwargs: additional arguments to pass to the exporter
+
+        Returns:
+            the XML element
+        """
+        p_idx = self.page_idx
+        block_count: int = 1
+        line_count: int = 1
+        word_count: int = 1
+        width, height = self.dimensions
+        language = self.language if 'language' in self.language.keys() else 'en'
+        # Create the XML root element
+        page_hocr = ETElement('html', attrib={'xmlns': 'http://www.w3.org/1999/xhtml', 'xml:lang': str(language)})
+        # Create the header / SubElements of the root element
+        head = SubElement(page_hocr, 'head')
+        SubElement(head, 'title').text = file_title
+        SubElement(head, 'meta', attrib={'http-equiv': 'Content-Type', 'content': 'text/html; charset=utf-8'})
+        SubElement(head, 'meta', attrib={'name': 'ocr-system', 'content': f"python-doctr {doctr.__version__}"})
+        SubElement(head, 'meta', attrib={'name': 'ocr-capabilities',
+                                         'content': 'ocr_page ocr_carea ocr_par ocr_line ocrx_word'})
```

Comment:
do we need to prepend `ocr_` to all element classes?

Reply:
@fg-mindee 
yes this follows the hOCR convention [convention](https://github.com/kba/hocr-spec/blob/master/1.1/spec.md)

Reply:
Sweet, could you add a link to this convention as a comment in the code please? :pray:

Reply:
Yes will do